### ISSUE TITLE
Text in activity box in external swimlane is hidden

### ISSUE THREAD

**Issue by JohnChrJensen** [ShareProc/shareproc-client-js#3]
![text in external swimlane is hidden](https://cloud.githubusercontent.com/assets/17935851/14150719/b00b433c-f6aa-11e5-83f5-814dcbecfb77.PNG)

When a new swimlane is created and an activity box with text is made, the text disappears. Most likely a layer issue.

**Comment by JohnChrJensen** [comment]
I found a workaround on this! 
By creating an activity box from an existing activity in the the existing pool and then drag it into the external pool, the text will be visible. It is still a bug, but much less critical with this workaround.

**Comment by baardl** [comment]
Very good you found a workaround.
Both dataObject and activity have these issues when pulling a new from left
pallette.
8. apr. 2016 9.53 a.m. skrev "JohnChrJensen" notifications@github.com:

**Comment by JohnChrJensen** [comment]
Yes, the workaround is therefore not working with the dataobject since you can not generate this from an existing object.
